src/libpmem2/persist.c

```diff
@@ -322,7 +322,7 @@ pmem2_deep_flush_byte(struct pmem2_map *map, void *ptr, size_t size)
 	LOG(3, "map %p ptr %p size %zu", map, ptr, size);
 
 	pmem2_persist_cpu_cache(ptr, size);
```

Comment:
Isn't this redundant now? `msync` will take care of flushing cpu caches.
But what about devdax?...